cypress/support/pages/teams/pageCommonTeams.js

```diff
@@ -0,0 +1,76 @@
+'use strict'
+
+class Teams_Common {
+
+  static #teamTitle = '[data-testid="title_of_an_element"]'
+  static #teamSettings = '[data-testid="team_settings"]'
+  static #manageTeamMembers = '[data-testid="administrate_team_members"]'
+  static #editTeam = '[data-testid="edit_team_members"]'
+  static #deleteTeam = '[data-testid="delete_team_members"]'
+  static #rocketchat = '[data-testid="rocketchat_wrapper"]'
+  static #teamOptions = '[data-testid="team_options"]'
+  static #activateRCCheckbox = '[data-testid="rocketchat_checkbox"]'
+  static #activateConfCheckbox =  '[data-testid="videoconf_checkbox"]'
+  static #teamsOverviewNavigationButton = '[data-testid="Teams"]'
+  static #messangerActivation = '[data-testid="rocketchat_checkbox"]'
+  static #saveButton = '[data-testid="create_team_btn"]'
+
+  navigateToTeamsOverview() {
+    cy.visit('/teams')
+    cy.get(Teams_Common.#teamsOverviewNavigationButton).click()
+    cy.url().should('include', '/teams')
+  }
+
+  selectTeam() {
```

Comment:
Do we here select the first team in the team overview?
Shouldn't we select a defined team if someone added one and the position is changed?

Reply:
yes, we need to stabilize the data passing in the test, also added in the follow up (#2039)